dotnet-8.0.404.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: dotnet-8.0.404
+  version: 8.0.404
+  epoch: 1
+  description: ".NET SDK"
+  copyright:
+    - license: MIT
+  resources:
+    cpu: 2
+    memory: 32Gi
+  dependencies:
+    runtime:
+      - dotnet-8-sdk-default
+      - icu
+
+environment:
+  environment:
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    DISABLE_CROSSGEN: true
+    DisableArcade: 1
+  contents:
+    packages:
+      - bash-binsh
+      - brotli-dev
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - clang-15
+      - clang-15-default
+      - cmake
+      - curl
```

Comment:
does dotnet need curl to work?

Reply:
these are build-dependencies; potentially we can trim them; it is likely that nuget uses curl.